packages/wxt/src/core/create-server.ts

```diff
@@ -162,16 +167,30 @@ async function createServerInternal(): Promise<WxtDevServer> {
       // Build after starting the dev server so it can be used to transform HTML files
       server.currentOutput = await internalBuild();
     } catch (err) {
-      if (!isBabelSyntaxError(err)) {
+      let watchFile: string | undefined;
+      // TODO: support syntax errors from ESBuild and others.
+      if (err instanceof SyntaxError && isBabelSyntaxError(err)) {
+        logBabelSyntaxError(err);
+        wxt.logger.info('Waiting for syntax error to be fixed...');
+        watchFile = err.id;
+      } else if (err instanceof Error && isModuleNotFoundError(err)) {
+        const details = parseModuleNotFoundError(err);
+        if (details && details.importer.startsWith('/@fs/')) {
+          wxt.logger.error(err.message);
+          wxt.logger.info('Waiting for import specifier to be fixed...');
+          watchFile = details.importer.slice(5);
```

Comment:
This isn't *perfect*, since the broken import may not be the fault of the importer. For example, it could be a configuration issue with `tsconfig.json` or something.

But it's not a big deal, IMO, since the developer can restart the `dev` process or save the importer's file if necessary.

Reply:
Before this can be merged, we should figure out how we can allow for saving the `wxt.config.ts` file to recreate the dev server as expected. Currently, it's just ignored, which is not a good DX.